frontend/src/components/Tournament.js

```diff
@@ -144,147 +185,153 @@ const Tournament = () => {
         icon={trophy}
         pageList={[{ url: "/tournaments", name: "All Tournaments" }]}
       />
+      <Suspense fallback={<TournamentPageSkeleton />}>
+        {/* Tournament title and status badge */}
+        <div class="px-1 py-3">
+          <div class="mb-4 flex flex-col gap-2">
+            <div>
+              <Switch>
+                <Match when={tournamentQuery.data?.status === "COM"}>
+                  <span class="h-fit rounded bg-blue-100 px-2.5 py-1 text-sm font-medium text-blue-800 dark:bg-blue-900 dark:text-blue-300">
+                    Completed
+                  </span>
+                </Match>
+                <Match when={tournamentQuery.data?.status === "LIV"}>
+                  <span class="h-fit rounded bg-green-100 px-2.5 py-1 text-sm font-medium text-green-800 dark:bg-green-900 dark:text-green-300">
+                    Live
+                  </span>
+                </Match>
+              </Switch>
+            </div>
+            <h1 class="text-pretty text-2xl">
+              <span class="w-fit bg-gradient-to-r from-blue-500 to-green-500 bg-clip-text font-extrabold text-transparent">
+                {tournamentQuery.data?.event?.title}
+              </span>
+            </h1>
+          </div>
 
-      <h1 class="mb-5 text-center">
-        <span class="w-fit bg-gradient-to-r from-blue-500 to-green-500 bg-clip-text text-2xl font-extrabold text-transparent">
-          {tournamentQuery.data?.event?.title}
-        </span>
-      </h1>
+          {/* Tournament image or date+location */}
+          <Show
+            when={tournamentQuery.data?.logo_dark}
+            fallback={
+              <LocationAndDate
+                location={tournamentQuery.data?.event?.location}
+                startDate={tournamentQuery.data?.event?.start_date}
+                endDate={tournamentQuery.data?.event?.end_date}
+              />
+            }
+          >
+            <div class="flex justify-start">
```

Comment:
The image is taking 3/4 of container width no ?

Reply:
ahh cool.. Then lets make it full width of the container? That should be good i think

Reply:
if we center it, it would look off on larger screens right ? everything else if left aligned regardless of screensize. Also, the image being off center might be because the beach nats image has a transparent border ?

| Beach nats | Surat |
| ---------- | ------------- |
| ![image](https://github.com/india-ultimate/hub/assets/39093537/1c45e0de-0872-4863-a42b-4e80ac5c259c) |![image](https://github.com/india-ultimate/hub/assets/39093537/ae157987-f592-4b4c-ae42-d89feea97b15) |

| left-align | center-align |
| --------- | ------- |
| ![image](https://github.com/india-ultimate/hub/assets/39093537/4b481b64-ad69-4a03-a666-68e98f85eb78) | ![image](https://github.com/india-ultimate/hub/assets/39093537/ab76c99f-4761-4a5c-aa9d-34b8400f608d) |

Reply:
hmm i still like full width more.. yes for larger screens we have to think diff. But what i meant before was image left aligned is not looking too good since its a square. Would be perfect if it was rectangle and didnt have any space left on the right. But since its square, not centering it looks off. Like I havent seen image left aligned in any other websites with space left on the right

Reply:
correct, the square image looks a bit off when left centered. I was talking about left aligning for larger screens only. Agreed.

Reply:
> hmm i still like full width more.. yes for larger screens we have to think diff. But what i meant before was image left aligned is not looking too good since its a square. Would be perfect if it was rectangle and didnt have any space left on the right. But since its square, not centering it looks off. Like I havent seen image left aligned in any other websites with space left on the right

hmm, maybe we should enforce a 4/3 or 16/9 aspect ratio then ?